app/components/task/modal.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if @showModal}}
+    <div class="overlay">
+        <div class="modal">
+            <button
+                type="button"
+                class="close"
+                {{on "click" @goBack}}
+            >
+                <img src="close.png" alt="close" height="20px" width="20px" />
```

Comment:
Can we please use the icon from these installed libraries?
![image](https://user-images.githubusercontent.com/78433013/190915828-d0cd51be-3b62-44cb-b9c8-bcf0c061e237.png)